src/e2e/app/cli/inmem/tests/tests/rest_api/test_swagger.rs

```diff
@@ -13,30 +13,30 @@ use kamu_cli_e2e_common::prelude::*;
 
 kamu_cli_run_api_server_e2e_test!(
     storage = inmem,
-    fixture = kamu_cli_e2e_repo_tests::test_login_password_predefined_successful,
+    fixture = kamu_cli_e2e_repo_tests::rest_api::test_swagger_page_st,
```

Comment:
Comment for readers:

Note: git[hub] got a little confused incorrectly displaying the difference -- it's not a moved and updated file, just a new one.